### ISSUE TITLE
Need some StackOverflow upvotes

### ISSUE THREAD

**Issue by dandv** [MeteorCommunity/chat#8]
[Sortable](http://stackoverflow.com/questions/20424477/responsive-sortable-list-supporting-drag-drop-for-bootstrap/27221306#27221306), the first 3rd party library with official Meteor integration, needs some upvotes on StackOverflow. CC @rubaxa, @MeteorCommunity/owners

**Comment by paralin** [comment]
Couldn't this be construed as vote manipulation?

**Comment by dandv** [comment]
Of course, please only vote if you like the quality of the answer, such as, I don't know the fact that it actually includes a working code snippet? :-]
